hazelcast/src/test/java/com/hazelcast/internal/cluster/impl/MembershipFailureTest.java

```diff
@@ -71,8 +70,8 @@
 import static org.junit.Assert.assertTrue;
 
 @RunWith(Parameterized.class)
-@Parameterized.UseParametersRunnerFactory(HazelcastParametersRunnerFactory.class)
-@Category({QuickTest.class, ParallelTest.class})
+@Parameterized.UseParametersRunnerFactory(HazelcastParallelParametersRunnerFactory.class)
+@Category({QuickTest.class})
```

Comment:
This is what we came up with as conclusion of the intermittent test failure: remove one layer of parallelization (the one on the class level).

Reply:
Yes, this test will not be a `ParallelTest`, but will run methods in parallel.